hub.yml

```diff
@@ -1,16 +1,16 @@
-name: policy-name
-description: DESCRIPTION OF YOUR POLICY
-homepage: POLICY HOMEPAGE URL
+name: psp-hostpaths
+description: PSP that controls usage of hostPath volumes
+homepage: https://github.com/kubewarden/psp-hostpaths-policy
 author:
-  name: policy-authors
-  homepage: https://author1.website
+  name: Kuberwarden devs
```

Comment:
I think we are using this value for all the policies we created. I think it gives them a sense of "trusted"

Reply:
So then it's fine to have different values on policy metadata vs policy hub?

Reply:
Agree with having `Kubewarden devs` on `hub.yml`. I put my personal name on `metadata.yml` because is what I saw from previous policies, but I think it would be better to have `Kubewarden devs` there, too.

Reply:
Done in https://github.com/kubewarden/psp-hostpaths-policy/pull/1/commits/cb83934f7af4282de84232edb2f5843292464aac.